src/Tests/Microsoft.TemplateEngine.Cli.UnitTests/ParserTests/Approvals/HelpTests.CanShowCommandOptions_Basic.verified.txt

```diff
@@ -1,8 +1,8 @@
 ﻿Options:
-  -n, --name <name>      The name for the output being created. If no name is specified, the name of the output directory is used.
-  -o, --output <output>  Location to place the generated output.
-  --dry-run              Displays a summary of what would happen if the given command line were run if it would result in a template creation.
-  --force                Forces content to be generated even if it would change existing files.
-  --no-update-check      Disables checking for the template package updates when instantiating a template.
-  --project <project>    The project that should be used for context evaluation.
```

Comment:
for these kinds of diffs, we should supply and HelpName if we want to keep this layout, correct @adamsitnik?

Reply:
> for these kinds of diffs, we should supply and HelpName if we want to keep this layout, correct

That is correct. Would you like me to add the `HelpName` values and keep the old layout?